Dockerfile

```diff
@@ -12,10 +12,32 @@ RUN curl -sL https://deb.nodesource.com/setup_6.x | bash \
     && apt-get install --no-install-recommends -y nodejs \
     && rm -rf /var/lib/apt/lists/*
 
+# Kosmtik with plugins
 RUN npm install -g kosmtik
 
+WORKDIR /usr/lib/node_modules/kosmtik/
```

Comment:
I've just checked that without WORKDIR `kosmtik plugins --install pluginname` uses filesystem root (`/`) and it's not working too, of course. Maybe Kosmtik should have some different plugins discovery/configuration method, because currently global npm installation doesn't work with plugins - there are two different ways to do it, for example:

`node index.js plugins --install kosmtik-place-search`

and 

`npm i kosmtik-place-search`

(on https://www.npmjs.com/package/kosmtik-place-search) and both are broken.